<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="aec491d7-9536-4bb2-a947-f189a1e39737">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/Simplechatbot/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Python_Projects\Simplechatbot\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
    <data-source source="LOCAL" name="db.sqlite3" uuid="38b8f9cc-cf85-4e22-8ef0-a798b4f760a0">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
  </component>
</project>